meta-android/recipes-core/images/initramfs-android-image.bb

```diff
@@ -8,9 +8,9 @@ ANDROID_EXTRA_INITRAMFS_IMAGE_INSTALL ?= ""
 
 VIRTUAL-RUNTIME_android-initramfs-scripts ?= "initramfs-scripts-android"
 
-IMAGE_INSTALL = "busybox base-passwd android-tools bash"
-IMAGE_INSTALL += "${VIRTUAL-RUNTIME_android-initramfs-scripts}"
-IMAGE_INSTALL += "${ANDROID_EXTRA_INITRAMFS_IMAGE_INSTALL}"
+IMAGE_INSTALL = "busybox busybox-mdev base-passwd android-tools bash"
+IMAGE_INSTALL += " ${VIRTUAL-RUNTIME_android-initramfs-scripts}"
```

Comment:
The leading space doesn't need to be added.

Reply:
@shr-project https://github.com/shr-distribution/meta-smartphone/pull/144/commits/0cc1008eaba923964d6dd401abf04ab7be9cc466